.gitignore

```diff
@@ -73,7 +73,8 @@ instance/
 .scrapy
 
 # Sphinx documentation
-docs/_build/
+_build/
```

Comment:
Should this be `site`? With `mkdocs build,` the `site` folder is created.

Reply:
the mkdocs site folder is lower in the .gitignore file.